### ISSUE TITLE
auto-value-cursor 2.0.0 does not include shadow

### ISSUE THREAD

**Issue by ganadist** [gabrielittner/auto-value-cursor#108]
When using auto-value-cursor 2.0.0, it raise following error

```
  Caused by: java.lang.ClassNotFoundException: com.gabrielittner.auto.value.util.Property
  	at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
  	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
  	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
```

**Comment by laurencedawson** [comment]
Also getting this

**Comment by gabrielittner** [comment]
Sorry for this, will look into it in the coming week. As a workaround you should be able to include a dependency on `com.gabrielittner.auto.value:auto-value-extension-util:0.4.0` using `annotationProcessor`/`kapt`.

**Comment by gabrielittner** [comment]
Released 2.0.1 to fix the issue
